tee-worker/service/src/cli.yml

```diff
@@ -122,6 +126,11 @@ subcommands:
                   long: skip-ra
                   short: s
                   help: skip remote attestation. Set this flag if running enclave in SW mode
+            - linkable:
+                long: linkable
+                short: l
+                help: Set this flag if this SPID is associated with linkable quotes.
+
```

Comment:
duplicate code block.

Reply:
those codes are no dup, this `linkable` arg is the args of `request-state` command, not for `run`.

Reply:
oh, yes. My mistake.